Add build platform configurations for osx and linux

runette opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

runette committed:
Add build platform configurations for osx and linux

runette committed:
Increment build number from 2 to 3

runette commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.24.23.37.0

runette committed:
Add python and pip to the build requirements

runette committed:
Update Python version requirement for build skip

runette committed:
Bump version from 1.1.0 to 1.1.1

runette committed:
Update build dependencies in meta.yaml

runette committed:
Modify meta.yaml for Python 3.11 and dependencies

Updated Python version requirement and adjusted dependencies for cross-platform compatibility.
